### DISCUSSION TITLE
Pause video for x seconds when an annotation starts

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#980] [Ideas]
In video annotations, I sometimes add a box in a single frame and not across multiple frames (meaning there is no track). So the annotation exists in just one frame of the video (I imagine). It is probably because the box's life is just 1 frame, when I play the video in normal speed or even slowed down, the box never shows up on the video. In fact if I do not look at the timeline section at the bottom I would not even notice that there are annotations present. I'd have to click on the individual annotations to see what they are etc. 
And this takes time especially when doing review work. It would be nice it there is an option I can turn on that automatically pauses for x amount of seconds when it comes across an annotation and then continues. This way I can review the annotations present only in one frame without having to click on each one and wait for it to load. This takes a lot of time. I explained the details on this in https://github.com/orgs/biigle/discussions/973

**Comment by mzur** [comment]
Before we discuss your proposed feature, could a "Volume Label Review" annotation mode for videos also be a solution for your workflow (see my [other comment](https://github.com/orgs/biigle/discussions/973#discussioncomment-11384010))?
